run.py

```diff
@@ -69,7 +67,8 @@ def install_secret_keys(application, secret_session='secret_key', secret_csrf='s
         sys.exit(1)
 
 
-install_secret_keys(app)
+if 'TESTING' not in os.environ or os.environ['TESTING'] == 'False':
```

Comment:
I'm unsure if this will work, because if the secret keys aren't installed, they default to None in Flask, which is not a good thing (I'm unsure what the CSRF validation will do at that point)